src/components/LoanCards/HoverLoanCard/HoverLoanCard.vue

```diff
@@ -170,12 +177,22 @@ export default {
 		},
 		handleClick() {
 			if (!this.hoverEffectActive()) {
-				this.$emit('update-detailed-loan-index', this.loanIndex);
+				this.updateDetailedLoanIndex();
 			}
 		},
 		trackInteraction(args) {
 			this.$emit('track-interaction', args);
 		},
+		hoverCardLargeUpdateDetailedLoanIndex() {
+			if (!this.rowHasDetailedLoan) {
+				this.$emit('set-prevent-updating-detailed-card', true);
+			}
+			this.updateDetailedLoanIndex();
+		},
+		hoverCardSmallUpdateDetailedLoanIndex() {
+			this.$emit('set-prevent-updating-detailed-card', false);
+			this.updateDetailedLoanIndex();
```

Comment:
2B) User explicitly clicks on a neighboring loan
  * This re-enables the mouseenter behavior